openhtf/io/output/console_summary.py

```diff
@@ -0,0 +1,107 @@
+"""Module to display test summary on console."""
+
+import os
+import sys
+import __main__ as main
+
+from openhtf import util
+from openhtf.io import test_record
+from openhtf.util import measurements as measurement
+
+
+class ConsoleSummary():
+  """Print test results with failure info on console. """
+
+  # pylint: disable=invalid-name
+  def __init__(self):
```

Comment:
can you take an `output_stream` argument here, default it to `sys.stdout`, and use it below instead of using `sys.stdout` directly?